config_sample.py

```diff
@@ -2,7 +2,7 @@
 # install.sh, the config.py should have been generated for you.
 APPLICATION_ROOT = None
 CSRF_ENABLED = True
-DATABASE_URI = 'mysql+pymysql://root:@localhost:3306/test'
+DATABASE_URI = 'mysql+pymysql://root:@localhost:3306/test?charset=utf8'
```

Comment:
Is there a reason this needs to be appended? I'd need to know to update the live config :)

Reply:
while listing users in User Manager, it created some encoding issue while reading db, so i placed utf8 charset there.